pootle/templates/editor/units/edit.html

```diff
@@ -13,6 +13,13 @@
       </ul>
     </div>
     <div class="translate-{% locale_align %}">
+      <!-- Priority -->
```

Comment:
I think this coupling is not problematic, but as I understand this problem can grow bigger in the future. So I would like this to be merged and deal with this problem in the future if it really gets to become something annoying.